xapian-letor/tests/api_letor.cc

```diff
@@ -234,11 +227,10 @@ DEFINE_TESTCASE(createfeaturevectorthree, generated)
     return true;
 }
 
-
 DEFINE_TESTCASE(preparetrainingfileonedb, generated)
 {
     string db_path = get_database_path("apitest_listnet_ranker1",
-				       db_index_one_document);
+				      db_index_one_document);
```

Comment:
This change looks wrong to me - the github diff viewer confuses things a bit as the second line with a leading tab appears to have a one column shift to the left, but when viewed in an editor with the correct settings, the first `d` on the line above should align below the first `"` on the line above that.